src/projectM-sdl/pmSDL.cpp

```diff
@@ -30,12 +30,12 @@ void projectMSDL::audioInputCallbackS16(void *userdata, unsigned char *stream, i
     app->pcm()->addPCM16(pcm16);
 }
 
-SDL_AudioDeviceID projectMSDL::selectAudioInput(int count) {
+SDL_AudioDeviceID projectMSDL::selectAudioInput(int _count) {
     // ask the user which capture device to use
     // printf("Please select which audio input to use:\n");
     printf("Detected devices:\n");
-    for (int i = 0; i < count; i++) {
-        printf("  %i: 🎤%s\n", i, SDL_GetAudioDeviceName(i, true));
```

Comment:
:(

Reply:
Why the special character in printf before %s

Reply:
its just an emoji representing an audio input device. adds flair. not important 👨‍🍳